integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -901,6 +936,9 @@ fn test_resharding_v3_delayed_receipts_left_child() {
             ReceiptKind::Delayed,
         ))
         .allow_negative_refcount(true)
+        // TODO(resharding): test should work without changes to num_rpcs and track_all_shards
+        .num_rpcs(0)
```

Comment:
Maybe it works now, after setting `base_epoch_config.num_block_producer_seats` etc.
Similarly other todos below.

Reply:
Two more tests work now, which is better but the others still require some attention